R/input.R

```diff
@@ -0,0 +1,104 @@
+library(openxlsx)
+library(readODS)
```

Comment:
You need to document these imports inside `ADSPlanner-package.R` and then run `devtools::document()`, or else they won't work when the package is compiled. (Check out my PR)